README.rst

```diff
@@ -1,6 +1,33 @@
 Cloudant-Python
 ===============
 
+This Version is Deprecated
+--------------------------
+
+As of 13 October 2015, this development branch is deprecated in favor of `the new 
+development branch starting at version 2.0.0a1 <https://github.com/cloudant/python-cloudant>`_. 
+
+**The new version will introduce breaking changes. No attempt was made to follow the 
+API in 0.5.10.**
+
+This is the final version of this branch -- 0.5.10. 
+
+Please use the new library for your new projects and begin to migrate your old projects that have 
+used versions 0.5.10 and prior. 
+
+We will keep 0.5.10 as the latest stable version on PyPI until at least early 2016, at which time 
```

Comment:
fixed in https://github.com/gadamc/cloudant-python/commit/cf85ca89b1f7384ef4d9778aa511d47b6af1df05#diff-88b99bb28683bd5b7e3a204826ead112